### PR TITLE
Improve Comment for connectivity results

### PR DESCRIPTION
**Pull request by rcassani**

### CODE REVIEW

File: toolbox/connectivity/bst_connectivity.m [hunk 1]
```diff
@@ -745,6 +752,23 @@
...
+            % Search for trial tags and remove them
+            for iComment = 1 : length(listComments)
+                [~, tmpStrs] = regexp(listComments{iComment},'\(#.+\)','match','split');
+                listComments{iComment} = deblank([tmpStrs{:}]);
+            end
+        end
+        if (length(unique(listComments)) == 1)
+            FileMat.Comment = listComments{1};
+        else
+            FileMat.Comment = listComments{end};
```
Comment (rcassani) [inline toolbox/connectivity/bst_connectivity.m:769]: For consistency with other averaged files, the Comment can be extended as:
`Avg: Comment (#files)`

Comment (ftadel) [reply toolbox/connectivity/bst_connectivity.m:769]: Yes, this makes sense.

File: toolbox/connectivity/bst_connectivity.m [hunk 2]
```diff
@@ -745,6 +752,23 @@
     FileMat.Method    = OPTIONS.Method;
     FileMat.DataFile  = file_win2unix(DataFile);
     FileMat.nAvg      = nAvg;
+    % Comment if average comes from trials
+    if FileMat.nAvg > 1
+        listComments = Comment;
+        % If Comments are the same
+        if length(unique(listComments)) > 1
+            % Search for trial tags and remove them
```
[the remark this answers is not in the document]
Comment (ftadel) [reply toolbox/connectivity/bst_connectivity.m:759]: Yes? No?

**Comment by rcassani** [comment]
In response to: https://github.com/brainstorm-tools/brainstorm3/pull/520#discussion_r824499715 
@ftadel let's keep it with the regex only targeting trial numbers

**Pull request merged**